source/MaterialXView/Viewer.cpp

```diff
@@ -883,6 +901,8 @@ void Viewer::loadDocument(const mx::FilePath& filename, mx::DocumentPtr librarie
 {
     // Set up read options.
     mx::XmlReadOptions readOptions;
+    readOptions.desiredMajorVersion = 1;
+    readOptions.desiredMinorVersion = 38;
```

Comment:
We may not want to activate this feature yet for the MaterialX viewer, as it would cause documents saved from the viewer to become 1.38 documents.  For now, I think it's better for core MaterialX tools to maintain 1.37 standards when documents are loaded and saved, with only custom pipelines leveraging 1.38 features.

Reply:
Yes. That makes sense. I'll revert this.